features/size_validation.feature

```diff
@@ -62,24 +68,28 @@ Feature: Applicant uploads too large a file for their application
       | shf_application_company_number | shf_application_phone_number | shf_application_contact_email |
       | 5560360793                     | 031-1234567                  | applicant_2@random.com        |
     And I select "Groomer" Category
+    And I select files delivery radio button "upload_now"
     And I choose a file named "upload-just-under-limit.pdf" to upload
     When I click on t("shf_applications.new.submit_button_label")
     Then I should not see t("activerecord.errors.models.uploaded_file.attributes.actual_file_file_size.file_too_large")
     And I should see t("shf_applications.create.success", email_address: applicant_2@random.com)
 
-
+  @selenium
   Scenario: Existing application - Uploads a file that is too large
     Given I am logged in as "emma@happymutts.se"
     And I am on the "edit my application" page
     And I choose a file named "diploma_huge.pdf" to upload
+    And I select files delivery radio button "upload_now"
     When I click on t("shf_applications.edit.submit_button_label")
     Then I should see t("activerecord.errors.models.uploaded_file.attributes.actual_file_file_size.file_too_large")
     And I should not see t("shf_applications.update.success")
 
+  @selenium
   Scenario: Existing application - Uploads a file that is too large then uploads ok file
     Given I am logged in as "emma@happymutts.se"
     And I am on the "edit my application" page
     And I choose a file named "diploma_huge.pdf" to upload
+    And I select files delivery radio button "upload_now"
```

Comment:
(see comment above; this step should come before "I choose a file named ... to upload')